gcc/rust/rust-session-manager.h

```diff
@@ -290,6 +290,7 @@ struct Session
   /* This should really be in a per-crate storage area but it is wiped with
    * every file so eh. */
   std::string injected_crate_name;
+  std::unordered_map<std::string, std::string> extern_crates;
```

Comment:
Do you want to switch this to a sorted tree and then we can merge it?
```suggestion
  std::map<std::string, std::string> extern_crates;
```

Reply:
Rebased and changes :+1: